airbyte-integrations/connectors/source-help-scout/integration_tests/invalid_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "client_id": "DnGIBWcQvoLekpts3zWMGQasdapbVrd4Ka",
```

Comment:
just confirming: are these bogus values that have nothing to do with the real prod values? :)

Reply:
Yep, just changed a few digits